lib/rucio/tests/test_judge_repairer.py

```diff
@@ -305,29 +307,44 @@ def test_to_repair_a_rule_with_only_1_rse_whose_site_is_blocklisted(self):
 
         rse = rse_name_generator()
         rse_id = add_rse(rse, **self.vo)
-        update_rse(rse_id, {'availability_write': False})
         set_local_account_limit(self.jdoe, rse_id, -1)
-
         rule_repairer(once=True)  # Clean out the repairer
-        scope = InternalScope('mock', **self.vo)
-        files = create_files(4, scope, self.rse4_id, bytes=100)
-        dataset = 'dataset_' + str(uuid())
-        add_did(scope, dataset, DIDType.DATASET, self.jdoe)
-        attach_dids(scope, dataset, files, self.jdoe)
 
-        rule_id = add_rule(dids=[{'scope': scope, 'name': dataset}], account=self.jdoe, copies=1, rse_expression=rse, grouping='DATASET', weight=None, lifetime=None, locked=False, subscription_id=None, ignore_availability=True, activity='DebugJudge')[0]
-
-        assert(RuleState.STUCK == get_rule(rule_id)['state'])
-        rule_repairer(once=True)
-
-        # Stil assert STUCK because of ignore_availability:
-        assert(RuleState.STUCK == get_rule(rule_id)['state'])
```

Comment:
This behavior is altered, so the old test case would fail. Based on our issue discussion, it sounds like this is OK.